includes/class-piklist-shortcode.php

```diff
@@ -238,21 +238,26 @@ public static function shortcode($attributes, $content = '', $tag)
   {
     if (self::$shortcodes[$tag])
     {
-      foreach ($attributes as $attribute => $attribute_value)
-      {
-        if (stristr($attribute_value, '%'))
-        {
-          $attributes[$attribute] = stripslashes(rawurldecode($attribute_value));
-        }
-      }
+	  if (!empty($attributes))	{
+		  
+		  foreach ($attributes as $attribute => $attribute_value)
+		  {
+			if (stristr($attribute_value, '%'))
+			{
+			  $attributes[$attribute] = stripslashes(rawurldecode($attribute_value));
+			}
+		  }
+	  }
 
       ob_start();
       
       do_action('piklist_pre_render_shortcode', $attributes, self::$shortcodes[$tag]);
 
       if (self::$shortcodes[$tag]['render'])
       {
-        $attributes['content'] = $content;
+		if (isset($attributes['content'])) {
```

Comment:
I don't think this is really necessary. In fact, I'm not sure why the content is being added to the attributes at all. @kevinpmiller do you have any insight as to why this is happening? Seems like we could just axe this code altogether.